packages/forms/src/Testing/TestsForms.php

```diff
@@ -0,0 +1,160 @@
+            /** @phpstan-ignore-next-line  */
+            $this->assertFormExists($formName);
+
+            $livewire = $this->instance();
+            $livewireClass = $livewire::class;
+
+            /** @var ComponentContainer $form */
+            $form = $livewire->$formName;
+
+            /** @var ?Field $field */
+            $field = data_get($form->getFlatFields(true), $fieldName, null);
+
+            Assert::assertInstanceOf(
+                Field::class,
+                $field,
+                "Failed asserting that a field with the name [{$fieldName}] exists on the form with the name [${formName}] on the [{$livewireClass}] component."
+            );
+
+            return $this;
+        };
+    }
+
+    public function assertFormFieldIsDisabled(): Closure
+    {
+        return function (string $fieldName, string $formName = 'form'): static {
+            /** @phpstan-ignore-next-line  */
+            $this->assertFormFieldExists($fieldName, $formName);
+
+            $livewire = $this->instance();
+            $livewireClass = $livewire::class;
+
+            /** @var ComponentContainer $form */
+            $form = $livewire->$formName;
+
+            /** @var Field $field */
+            $field = $form->getFlatFields(true)[$fieldName];
+
+            Assert::assertTrue(
+                $field->isDisabled(),
+                "Failed asserting that a field with the name [{$fieldName}] is disabled on the form named [{$formName}] on the [{$livewireClass}] component."
+            );
+
+            return $this;
+        };
+    }
+
+    public function assertFormFieldIsEnabled(): Closure
+    {
+        return function (string $fieldName, string $formName = 'form'): static {
+            /** @phpstan-ignore-next-line  */
+            $this->assertFormFieldExists($fieldName, $formName);
+
+            $livewire = $this->instance();
+            $livewireClass = $livewire::class;
+
+            /** @var ComponentContainer $form */
+            $form = $livewire->$formName;
+
+            /** @var Field $field */
+            $field = $form->getFlatFields(true)[$fieldName];
+
+            Assert::assertFalse(
+                $field->isDisabled(),
+                "Failed asserting that a field with the name [{$fieldName}] is enabled on the form named [{$formName}] on the [{$livewireClass}] component."
+            );
+
+            return $this;
+        };
+    }
+
+    public function assertFormFieldIsHidden(): Closure
+    {
+        return function (string $fieldName, string $formName = 'form'): static {
+            /** @phpstan-ignore-next-line  */
+            $this->assertFormFieldExists($fieldName, $formName);
+
+            $livewire = $this->instance();
+            $livewireClass = $livewire::class;
+
+            /** @var ComponentContainer $form */
+            $form = $livewire->$formName;
+
+            $fields = $form->getFlatFields();
```

Comment:
Please explicitly pass `false` here instead of relying on the default, also using a named parameter

Reply:
This is updated now